client/login/wp-login/index.js

```diff
@@ -50,21 +50,38 @@ class Login extends React.Component {
 		}
 	}
 
+	goBack( event ) {
+		event.preventDefault();
+
+		if ( typeof window !== 'undefined' ) {
+			window.history.back();
```

Comment:
In some places we use also `page.back()`, but I'm not sure if that makes any difference.